src/readabilityScore/Text.java

```diff
@@ -0,0 +1,115 @@
+package readabilityScore;
+
+public class Text {
+    private final int words;
```

Comment:
It would be better to separate this into two classes: one to store the text statistics with getters and one to calculate the text statistics:
```
public class TextStatisticsCalculator {
    public TextStatistics calculate(String text) {
        ...
    }
}
```